src/Faker/Provider/es_MX/Person.php

```diff
@@ -0,0 +1,337 @@
+
+
+    public function lastName()
+    {
+        return static::randomElement(static::$lastName).' '.static::randomElement(static::$lastName);
+    }
+
+    public function lastNameMother()
+    {
+        return static::randomElement(static::$lastNames);
+    }
+
+    public function lastNameFather()
+    {
+        return static::randomElement(static::$lastNames);
+    }
+    /**
+     * generates CURP
```

Comment:
would this link suffice? https://es.wikipedia.org/wiki/Clave_%C3%9Anica_de_Registro_de_Poblaci%C3%B3n